src/x/retry/retry.go

```diff
@@ -188,3 +189,15 @@ func BackoffNanos(
 	}
 	return backoff
 }
+
+// histogramWithDurationBuckets returns a histogram with the standard duration buckets.
+func histogramWithDurationBuckets(scope tally.Scope, name string) tally.Histogram {
+	sub := scope.Tagged(map[string]string{
+		// Bump the version if the histogram buckets need to be changed to avoid overlapping buckets
+		// in the same query causing errors.
+		"schema": "v1",
+	})
+	buckets := append(tally.DurationBuckets{0},
+		tally.MustMakeExponentialDurationBuckets(time.Millisecond, 1.25, 30)...)
```

Comment:
Seems like 30 buckets is only up to 600ms visibility:
https://goplay.space/#62qDCvfoeJZ

I know @vdarulis suggested less than 60, with a slightly higher factor and using 30 gives visibility up to 4minutes:
https://goplay.space/#8bQLM-SYl2Q

Using:
```go
append(tally.DurationBuckets{0, time.Millisecond}, 
  tally.MustMakeExponentialDurationBuckets(2*time.Millisecond, 1.5, 30)...)
```